codespell_lib/data/dictionary.txt

```diff
@@ -55627,6 +55628,7 @@ trrigger->trigger
 trriggered->triggered
 trriggering->triggering
 trriggers->triggers
+tru->true
```

Comment:
Although uncommon:
```suggestion
tru->true, trug,
```
Perhaps even `through` as a fix for `thru`.

Reply:
Like 'foals', I find 'trug' very unlikely to be present in code, and even less misspelled.

I was not under the impression that it was the policy to have rare words in the dictionnary.

Again, I'll add that if confirmed that I should.

Reply:
Both [`foal`](http://app.aspell.net/lookup?words=foal) and [`trug`](http://app.aspell.net/lookup?words=trug) can be found in [SCOWLS](http://wordlist.aspell.net/), which I see as the reference for English spellchecking.

In this specific case, adding `tru->through` might be enough, because suggesting multiple words force users to give the proposed fixes a thought, and handle rare word themselves:
```suggestion
tru->true, through,
```